res/deploy.tpl.php

```diff
@@ -67,13 +76,15 @@ function () {
 task(
 	'wp:opcache_clear',
 	function () {
+		if ( ! get( 'wp_clear_opcache' ) ) {
```

Comment:
As we're running wp commands here, shouldn't we check for the new `wordpress` flag as well? For when `wp_clear_opcache` got forgotten to delete.
```suggestion
		if ( ! get( 'wp_clear_opcache' ) || ! get( 'wordpress' ) ) {
```

Reply:
Not sure about this, why would someone set `wp_clear_opcache` after `wordpress` is set to false?

Reply:
Just for when somebody forgets to set this to false. Same with `wp_languages` for example.

Reply:
You don't have to set it to false. The option should be omitted if not used.